src/bundle/Resources/public/js/CKEditor/plugins/elements-path.js

```diff
@@ -0,0 +1,60 @@
+import Plugin from '../../../../../../../../ezplatform-admin-ui-assets/Resources/public/vendors/@ckeditor/ckeditor5-core/src/plugin';
+
+class ElementsPath extends Plugin {
+    constructor(props) {
+        super(props);
+
+        this.elementsPathWrapper = null;
+
+        this.updatePath = this.updatePath.bind(this);
+        this.getElementsPathWrapper = this.getElementsPathWrapper.bind(this);
+    }
+
+    getElementsPathWrapper() {
+        if (!this.elementsPathWrapper) {
+            this.elementsPathWrapper = this.editor.sourceElement.parentElement.querySelector('.ez-elements-path');
```

Comment:
getter setting value? Nope nope nope, do it in constructor, didMount or sth like this